_data/retail.yml

```diff
@@ -648,6 +672,17 @@ websites:
   btc: true
   bch: true
   othercrypto: true
+- name: iT'S ONLY NATURAL
```

Comment:
Is this the proper capitalization?

Reply:
Unfortunately, yes

![image](https://user-images.githubusercontent.com/922233/39662360-83c9783e-508a-11e8-8dd6-6cbf46c70d82.png)